src/main.cpp

```diff
@@ -1,8 +1,7 @@
 /***************************************************************************
  *   Copyright (C) 2008-2013 by Heiko Koehn - KoehnHeiko@googlemail.com    *
- *   Copyright (C) 2013-2014, 2016 by Stephen Lyons                        *
- *                                            - slysven@virginmedia.com    *
- *   Copyright (C) 2014 by Ahmed Charles - acharles@outlook.com            *
+ *   Copyright (C) 2013-2016 by Stephen Lyons - slysven@virginmedia.com    *
```

Comment:
I didn't make any changes to this file in 2015, [so I deliberately didn't include it](https://www.gnu.org/licenses/gpl-howto.html)...! :open_mouth: